npm_and_yarn/helpers/run.js

```diff
@@ -10,7 +8,9 @@ const input = [];
 process.stdin.on("data", data => input.push(data));
 process.stdin.on("end", () => {
   const request = JSON.parse(input.join(""));
-  const func = functionMap[request.function];
+  const [manager, functionName] = request.function.split("_");
+  const helpers = require(`./lib/${manager}`);
+  const func = helpers[functionName];
```

Comment:
Could then keep the old functionality here:

```suggestion
  const func = functionMap[request.function];
```